solr/solr-ref-guide/src/common-query-parameters.adoc

```diff
@@ -307,11 +307,13 @@ The `echoParams` parameter controls what information about request parameters is
 
 The `echoParams` parameter accepts the following values:
 
-* `explicit`: This is the default value. Only parameters included in the actual request, plus the `_` parameter (which is a 64-bit numeric timestamp) will be added to the `params` section of the response header.
+* `explicit`: Only parameters included in the actual request, plus the `_` parameter (which is a 64-bit numeric timestamp) will be added to the `params` section of the response header.
```

Comment:
@gerlowskija this reminds me... (off topic of this PR), do you know where and why "_" parameter is added?  If it's for query identification purposes then it's duplicitous with the "rid" you added.  I'm having trouble searching for it.

Reply:
No, I don't know anything about it.  This is the first I've heard of an underscore param.

It may be duplicitous with the 'rid' stuff I added recently, but that's not necessarily the case.  e.g. There's nothing here that suggests that a '_' value gets passed without change to shard-requests, afaict.  But it's worth following up on for sure, thanks for pointing it out @dsmiley 

> I'm having trouble searching for it.
No kidding!  Whoever put this in didn't have searchability in mind, that's for sure haha.

Reply:
I did some digging and discovered that the admin UI will add this underscore.  See `services.js` which does this via `Date.now()` all over the place.  I don't know what purpose it has; git blame shows it was added with that whole UI refactor of it's day but perhaps it predated it?  AFAICT Solr isn't doing anything with it Solr side; I set multiple conditional debugger breakpoints across SolrParams subclasses but no where is "_" requested.  My suspicion is that it was added to foil HTTP cache attempts?

I did some JIRA digging and found: https://issues.apache.org/jira/browse/SOLR-4311 which has a patch with the underscores but it was for version, not the timestamp.  Given the underscore's use to defeat bad caching in that scenario, if I keep digging, I'll probably find it.

In short, I don't think we should document this underscore param in the ref guide.  It's not a special param to Solr; it's an oddity of our current admin UI.

Reply:
@arafalov can you please remove the '_' reference here in the docs?  I think what it says about this is false; I suspect it was always false.

Reply:
Will do. I was busy on the other project, but will finish this one very soon.